### PR TITLE
Ensure consistency between the radio button and checkbox

### PR DESCRIPTION
**Pull request by imisi-akande**
**What does this PR do?** 
- Ensure consistency between the radio button and checkbox on assessment pages, images
and display updated changes on the test-server 

**Description of the tasks** 
- Image displaying checkbox styles changed to radio buttons for consistency on page
style radio

- Image displaying NEXT text in button changed to NEXT LESSON and PREV to PREVIOUS LESSON

button-text

Changes can be viewed directly on this URL - https://cforafrica-mooc-test.appspot.com/technology/unit?unit=2&lesson=5

### CODE REVIEW

File: coursebuilder/modules/explorer/_static/css/style.css [hunk 1]
```diff
@@ -3594,7 +3594,7 @@ footer .footer-bottom .footer-social-icons a { color: black; }
 
 .course-card { position: relative; }
 
-.course-card img { width: 100%; }
+.course-card img { width: 100%; max-height: 400px; }
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/css/style.css]: @andela-iakande max-width: 100% and not width: 100%

Comment (imisi-akande) [reply coursebuilder/modules/explorer/_static/css/style.css]: @chege done.

**Comment by chegejames** [comment]
@andela-iakande Are the changes here only those for image sizing?

**Comment by imisi-akande** [comment]
@chegejames No, It also contains styling checkbox to radio button for consistency purpose recommended by Adams Oxford. Since @Tawakalt is now handling the image resizing I pegged it to what I did last on the feature.

**Comment by chegejames** [comment]
@andela-iakande I only see three https://github.com/CodeForAfricaLabs/coursebuilder-core/pull/37/files all with changes to images but none for the radio button.

**Comment by imisi-akande** [comment]
@chegejames so sorry about that. I missed it on cherrypicking. It is there now.

**Pull request merged**